test/core/rules/test_cases/L046.yml

```diff
@@ -0,0 +1,10 @@
+rule: L046
+
+test_simple:
+  pass_str: SELECT 1 from {{ ref('foo') }}
+
+test_fail_1:
+  fail_str: SELECT 1 from {{ref('foo')}}
```

Comment:
How fix-able do you think this rule is?

Reply:
In theory easy, but I don't think we have a good mechanism for fixing the source file rather than the templated file. Could be a useful extension in future, but I'm not going to tackle it right now.